metrics/ApplyMonitorDefArgs/postgres.snap

```diff
@@ -9,7 +9,7 @@ select
   DATE_TRUNC('HOUR', ingested_at) + '1 HOUR' as time_segment
 from default.runs 
 
-group by time_segment
+group by DATE_TRUNC('HOUR', ingested_at) + '1 HOUR'
```

Comment:
why not `group by time_segment`?

Reply:
to put it differently, if postgres support using derived column it should be handled in the dialect, if it doesn't support it it should be like here

Reply:
this was solved by dialect, why it is not now?

Reply:
it wasn't handled by dialect, now it is

Reply:
example of BigQuery https://github.com/getsynq/cloud/pull/12950/files where dialect is using expression instead of alias